app/lib/csv/buyers_exporter.rb

```diff
@@ -24,10 +25,10 @@ def values_for_account(account)
       account.bought_account_plan.try!(:name),
       account.created_at.to_s(:db),
       account.bought_cinstances.count,
-      account.first_admin.try!(:display_name),
-      account.first_admin.try!(:email),
+      account.decorate.admin_user_display_name,
+      user&.email,
```

Comment:
`user` is not nil, it is either a record or a User.new

Reply:
`admin_user` can return `nil`, this is why before this PR it was doing `try!(...`.

At first I did `user = (account.admin_user || User.new).decorate` but @guicassolato suggested that this could lead to wrong values:
https://github.com/3scale/porta/pull/2024#discussion_r451386790

I've recently added a test for this scenario in https://github.com/3scale/porta/pull/2040